src/lsptoolshost/roslynLanguageServer.ts

```diff
@@ -518,14 +518,19 @@ export class RoslynLanguageServer {
                 _channel.appendLine('Activating C# + C# Dev Kit...');
             }
 
+            // Set command enablement to use DevKit commands.
+            vscode.commands.executeCommand('setContext', 'dotnet.server.activationContext', 'RoslynDevKit');
```

Comment:
Does the place this is being set in this function matter specifically? i.e. this is after we've activated DevKit I think, but before we've done some other stuff? I think my request here is if order does matter, stick a comment explaining why you chose this spot.

Reply:
Order isn't super important - we're just setting it as soon as we know devkit is available (this is before its activated)